requirements.txt

```diff
@@ -0,0 +1,17 @@
+aiohttp==3.7.3
+aiohttp-debugtoolbar==0.6.0
+aiohttp-jinja2==1.4.2
+async-timeout==3.0.1
+attrs==20.3.0
+certifi==2020.11.8
+chardet==3.0.4
+idna==2.10
+Jinja2==2.11.2
+MarkupSafe==1.1.1
+multidict==5.0.2
+pkg-resources==0.0.0

```

Comment:
```
(venv) MacBook-Pro-kirill:mocksha kirill$ python -m aiohttp.web -H localhost -P 8080 mocksha
/Users/kirill/Prog/mocksha/venv/bin/python: Error while finding module specification for 'aiohttp.web' (ModuleNotFoundError: No module named 'aiohttp')
(venv) MacBook-Pro-kirill:mocksha kirill$ python -m pip install -r requirements.txt 
Collecting aiohttp==3.7.3
  Using cached aiohttp-3.7.3-cp39-cp39-macosx_10_14_x86_64.whl (648 kB)
Collecting aiohttp-debugtoolbar==0.6.0
  Using cached aiohttp_debugtoolbar-0.6.0-py3-none-any.whl (465 kB)
Collecting aiohttp-jinja2==1.4.2
  Using cached aiohttp_jinja2-1.4.2-py3-none-any.whl (11 kB)
Collecting async-timeout==3.0.1
  Using cached async_timeout-3.0.1-py3-none-any.whl (8.2 kB)
Collecting attrs==20.3.0
  Using cached attrs-20.3.0-py2.py3-none-any.whl (49 kB)
Collecting certifi==2020.11.8
  Using cached certifi-2020.11.8-py2.py3-none-any.whl (155 kB)
Collecting chardet==3.0.4
  Using cached chardet-3.0.4-py2.py3-none-any.whl (133 kB)
Collecting idna==2.10
  Using cached idna-2.10-py2.py3-none-any.whl (58 kB)
Collecting Jinja2==2.11.2
  Using cached Jinja2-2.11.2-py2.py3-none-any.whl (125 kB)
Collecting MarkupSafe==1.1.1
  Using cached MarkupSafe-1.1.1.tar.gz (19 kB)
Collecting multidict==5.0.2
  Using cached multidict-5.0.2-cp39-cp39-macosx_10_14_x86_64.whl (48 kB)
ERROR: Could not find a version that satisfies the requirement pkg-resources==0.0.0 (from -r requirements.txt (line 12)) (from versions: none)
ERROR: No matching distribution found for pkg-resources==0.0.0 (from -r requirements.txt (line 12))
WARNING: You are using pip version 20.2.3; however, version 20.3 is available.
You should consider upgrading via the '/Users/kirill/Prog/mocksha/venv/bin/python -m pip install --upgrade pip' command.
```

Reply:
According to https://github.com/pypa/pip/issues/4022, this is a bug resulting from Ubuntu providing incorrect metadata to pip.
`pip uninstall pkg-resources==0.0.0`
updated requirements.txt